web/src/test/java/com/replica/RouterServerTest.java

```diff
@@ -237,11 +238,11 @@ public void testAccessEgressCustomModesWithTransfer() {
         expectedModeCounts.put("car", 1);
         expectedModeCounts.put("foot", 2);
 
-        // Expected route is [car access -> transit -> transfer -> transit -> transit (no transfer) -> walk egress]
-        checkTransitQuery(response, 3, 3,
+        // Expected route is [car access -> transit -> transfer -> transit -> walk egress]
```

Comment:
The routing behavior with `car` access mode changed slightly in this unit test, so I had to change the destination to get a route returned that included a transfer.

Because none of the knobs used for this request actually changed (we went from a `beta_walk_time` of 1.5 to beta access/egress knobs set to 1.5), I'm guessing it was due to the fixed auto snapping detail that @michaz  added in graphhopper master recently